test/functions_test.rb

```diff
@@ -87,14 +103,14 @@ def test_functions_may_accept_sass_color_type
     end
 
     def test_function_with_unsupported_tag
-      engine = Engine.new("div {url: function_with_unsupported_tag(1);}")
+      engine = Engine.new("div {url: function_with_unsupported_tag(());}")
 
       exception = assert_raises(SassC::SyntaxError) do
         engine.render
       end
 
-      assert_match /Sass argument of type sass_number unsupported/, exception.message
-      assert_equal "[SassC::FunctionsHandler] Sass argument of type sass_number unsupported", stderr_output
+      assert_match /Sass argument of type sass_list unsupported/, exception.message
+      assert_equal "[SassC::FunctionsHandler] Sass argument of type sass_list unsupported", stderr_output
```

Comment:
Can't test unsupported tags with numbers anymore, seeing as we're now supporting numbers.